docs/src/app/components/pages/components/Popover/ExampleAnimation.jsx

```diff
@@ -48,7 +50,12 @@ export default class PopoverExampleAnimation extends React.Component {
           animation={PopoverAnimationFromTop}
         >
           <div style={styles.popover}>
```

Comment:
What about removing this node?
![capture d ecran 2016-02-08 a 12 42 00](https://cloud.githubusercontent.com/assets/3165635/12884853/5eab1dc6-ce61-11e5-8d66-c435d30d5e70.png)